### DISCUSSION TITLE
Please upload Cider on Steam

### DISCUSSION THREAD

**Discussion by Mabylife** [ciderapp/Cider#1757] [General]
I really want to buy Cider, but since that it is only uploaded on Microsoft store and I dont have a credit card so I cannot buy it.
Btw I have been using Cider classic for almost a year and I really love it :3

**Comment by GamingLiamStudios** [comment]
Unfortunately, our previous attempt at getting Cider over to Steam was rejected by Valve. However, we are now also on [Itch.io](https://cidercollective.itch.io/cider) with releases for Windows, MacOS and Linux, as well as a Portable release.
